docs/SupportProgress.md

```diff
@@ -0,0 +1,48 @@
+# The Operators and Functions Support Progress
+Although the Gluten project is still in active development, we still have made big progress on Operators and Functions support.
+- The number of Operators supported is 12, the most common Operators are covered.
+- The number of Functions supported is 94, and Velox supports 127 Functions.
```

Comment:
@lviiii @zhixingheyi-tian Would you like to help check the function part? Thanks.